Source/Core/Core/PowerPC/SignatureDB.h

```diff
@@ -31,9 +31,12 @@ class SignatureDB
   // Returns the hash.
   u32 Add(u32 startAddr, u32 size, const std::string& name);
 
-  bool Load(const std::string&
-                filename);  // Does not clear. Remember to clear first if that's what you want.
+  // Does not clear. Remember to clear first if that's what you want.
+  bool Load(const std::string& filename);
+  bool LoadAsCSV(const std::string& filename);
```

Comment:
Just an idea, but it might be more beneficial to make an interface for the loading and saving of arbitrary file formats, like:

```cpp
class SignatureDBFormatHandler
{
public:
    // Implement in cpp file, to prevent the vtable being placed
    // in every translation unit that uses this class.
    virtual ~SignatureDBFormatHandler();

    bool Load(const std::string& file_path, SignatureDB::FuncDB& database) = 0;
    bool Save(const std::string& file_path, const SignatureDB::FuncDB& database) = 0;
};
```

which can then just be implemented to support arbitrary formats.